### PR TITLE
Submission UI

### PR DESCRIPTION
**Pull request by tsbolty**
### What this PR does (required):
- Created submission page structure and styling. No functionality

### Screenshots / Videos (required):
image

image

### Any information needed to test this feature (required):
- Currently I am using the "Discover" tab in the Auth Header to get to this page. This will need to change in the future.
- Will also need to pass in contestId to navigate to page.

### Any issues with the current functionality (optional):
- No issues

### CODE REVIEW

File: client/src/pages/SubmitDesign/useStyles.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    textAlign: 'center',
+  },
+  bottomLine: {
+    marginBottom: '2rem',
+  },
+  uploadBtn: {
+    backgroundColor: 'black',
+    border: '1px solid white',
+    color: 'white',
+    textTransform: 'uppercase',
```
Comment (rajivtitus) [inline client/src/pages/SubmitDesign/useStyles.ts:44]: 👍🏽

File: client/src/pages/SubmitDesign/useStyles.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  container: {},
```
Comment (rajivtitus) [inline client/src/pages/SubmitDesign/useStyles.ts]: Please remove empty styles :)

File: client/src/App.tsx [hunk 3]
```diff
@@ -20,6 +21,7 @@ function App(): JSX.Element {
               <Switch>
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
+                <Route exact path="/submitdesign/:id" component={SubmitDesign} />
```
Comment (rajivtitus) [inline client/src/App.tsx]: Please use hyphenated case when route names contain multiple words :)

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 4]
```diff
@@ -21,7 +21,9 @@ const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
       <Typography color="secondary" className={classes.tattooArt}>tattoo art</Typography>
       {loggedInUser ? (
         <Box display="flex" justifyContent="space-between" alignItems="center" minWidth="55vw" flexWrap="wrap" className={classes.linkContainer}>
-          <Typography className={classes.navLink} color="secondary" display="inline">Discover</Typography>
+          <Link to="/submitdesign/99">
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/AuthHeader.tsx]: Same as above!

**Review by rajivtitus — APPROVED** [review]
UI looks great! Good job 👍🏽 

Just one or two small comments, please have a look.

File: client/src/pages/SubmitDesign/useStyles.ts (hunk 2 above)
Comment (tsbolty) [reply client/src/pages/SubmitDesign/useStyles.ts]: This is done.

File: client/src/pages/SubmitDesign/useStyles.ts (hunk 1 above)
Comment (tsbolty) [reply client/src/pages/SubmitDesign/useStyles.ts:44]: Thank you

File: client/src/components/AuthHeader/AuthHeader.tsx (hunk 4 above)
Comment (tsbolty) [reply client/src/components/AuthHeader/AuthHeader.tsx]: Also done

File: client/src/App.tsx (hunk 3 above)
Comment (tsbolty) [reply client/src/App.tsx]: Done

**Pull request merged**